comps/comps-foreman-fedora24.xml

```diff
@@ -31,6 +31,7 @@
       <packagereq type="default">foreman-vmware</packagereq>
 
       <packagereq type="default">foreman-installer</packagereq>
+      <packagereq type="default">nodejs-jquery</packagereq>
```

Comment:
Move into a new section below like you've done with others, this bit contains installer deps.